Scenes/World/Layer.gd

```diff
@@ -46,11 +46,11 @@ func generate_tree() -> void:
 	add_child(tree)
 	
 	tree.position.x = -position.x + WINDOW_WIDTH + 200
-	tree.position.y = 550
+	tree.position.y = 520
```

Comment:
Why 520? Maybe:
```tree.position.y = TREE_EXTENT```
and above in the code to give:
```const TREE_EXTENT= 520```

Reply:
You are really right. It's not good practice to hard coding.